models/product-type/README.md

```diff
@@ -0,0 +1,25 @@
+# @commercetools-test-data/product-type
+
+This package provides the data model for the commercetools platform `ProductType` type
+
+https://docs.commercetools.com/api/projects/productTypes#product-types
+
+# Install
+
+```bash
+$ yarn add -D @commercetools-test-data/product-type
+```
+
+# Usage
+
+```ts
+import type {
+  TProductTypeDefault,
+  TProductTypeDraftDefault,
```

Comment:
I suspect consumers of this will pull either the REST or GraphQL representations, not the default (which has both, right)?

Reply:
Yes. Went overboard here. The purpose of the readme is to show an example of just the REST method?

Reply:
I think so?

Reply:
I get that impression, at least.